mantidproject/mantid
# nGEM Data Loader Created

**Description of work.**
A new loader has been created that takes `.edb` files generated by the nGEM detector.
The loader loads the data into an event workspace and has an option to generate a matrix workspace that contains the number of neutrons detected during each frame. 

**Report to:** Davide Raspino

**To test:**
Files for testing are available on the Babylon shared filesystem: `/Babylon/Public/Davide/nGEM/data`
1. Load the single file or the first of the multi-files. (Ensure "GenerateEventsPerFrame" is ticked.)
2. Check the instrument view and color fill plots work and check the bin widths are correct. 
3. Check the event counts workspace to determine a lower bound for the number of events.
4. Reload the data with the new lower bound in the "MinEventsPerFrame" field.
5. Check that the extra values are no longer in the event_counts workspace and the sample logs are different. 

Closes #24306 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
